sci-astronomy/stellarium/stellarium-24.1.recipe

```diff
@@ -99,7 +100,18 @@ INSTALL()
 		-e "s|@MINOR@|$MINOR|" \
 		-e "s|@LONG_INFO@|$LONG_INFO|" \
 		$portDir/additional-files/stellarium.rdef.in > stellarium.rdef
-
+	
```

Comment:
This will not please jarvis :) Trailing whitespace.

Reply:
Oops…